.github/workflows/unit_tests.yml

```
@@ -41,4 +41,49 @@ jobs:
     - name: Set up Hatch
       uses: pypa/hatch@a3c83ab3d481fbc2dc91dd0088628817488dd1d5
     - name: Run unit tests (with coverage report at the end)
-      run: hatch test -c -py ${{ matrix.python-version }}
+      run:  |
+        if [[ "${{ matrix.os }}" == "ubuntu-latest" && "${{ matrix.python-version }}" == "3.11" ]]; then
+          hatch test -c -py ${{ matrix.python-version }} > cov.txt
```

```suggestion
          hatch test -c -py ${{ matrix.python-version }} | tee > cov.txt
```

This way, we still print the ouput in the action and also in the file.